saleor/graphql/discount/types/sales.py

```diff
@@ -52,12 +47,8 @@ class SaleChannelListing(ModelObjectType[models.SaleChannelListing]):
 
     class Meta:
         description = "Represents sale channel listing."
-        model = models.SaleChannelListing
         interfaces = [relay.Node]
-
-    @staticmethod
-    def resolve_channel(root: models.SaleChannelListing, info: ResolveInfo):
-        return ChannelByIdLoader(info.context).load(root.channel_id)
```

Comment:
Why did we drop this?

Reply:
There will be no related `PromotionChannelListing` model, so we need to kind of imitate it.